app/models/katello/concerns/operatingsystem_extensions.rb

```diff
@@ -21,7 +21,7 @@ def assign_templates!
               provisioning_template_name = Setting["katello_default_#{kind.name}"]
             end
 
-            if (template = ProvisioningTemplate.find_by_name(provisioning_template_name))
+            if (template = ProvisioningTemplate.unscoped.find_by_name(provisioning_template_name))
```

Comment:
Confirmed with testing that at least Ptable also needs to be unscoped.  Others to check may be OsDefaultTemplate and TemplateKind.